x-pack/plugins/ml/public/application/services/elastic_models_service.test.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticModels } from './elastic_models_service';
+import { type TrainedModelsApiService } from './ml_api_service/trained_models';
+
+const getMockResponse = () => [
+  {
+    version: 1,
+    config: {
+      input: {
+        field_names: ['text_field'],
+      },
+    },
+    description: 'Elastic Learned Sparse EncodeR v1 (Tech Preview)',
+    name: '.elser_model_1',
+  },
+  {
+    version: 2,
+    default: true,
+    config: {
+      input: {
+        field_names: ['text_field'],
+      },
+    },
+    description: 'Elastic Learned Sparse EncodeR v2 (Tech Preview)',
+    name: '.elser_model_2',
+  },
+  {
+    version: 2,
+    os: 'linux',
+    arch: 'x86_64',
```

Comment:
If these need to match what the ES nodes info API returns then annoyingly they need to be different to our standard platform identifiers:

```suggestion
    os: 'Linux',
    arch: 'amd64',
```

But if these are displayed to users then we would want them how you've got them now, and have a secret mapping to the values the JVM returns somewhere.

Reply:
Updated in https://github.com/elastic/kibana/pull/167000/commits/2c1eb3562868c690c56a4ed450674bc10628e280